src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -611,6 +625,33 @@ describe("Editor Page Component", () => {
             editorPage = wrapper.find(EditorPage).childAt(0);
             expect(editorPage.state().lockedTags).toEqual([]);
         });
+
+        it("Clears locked tags when tag is clicked", async () => {
```

Comment:
add an identical one but for when hot-key selection is use

Reply:
So the way we are simulating the events (click simulations vs dispatching of key events) makes it pretty difficult to combine the two simulations. In order to set locked tags to begin with, I need to simulate a click event that sets the locked tags. I can't set them directly in state, because the editor page is not the root component. But I think it's adequate to show the selected tag is changing because those are being set by the same call to `setState`